MantidQt/MantidWidgets/src/WorkspacePresenter/QWorkspaceDockView.cpp

```diff
@@ -1705,5 +1687,11 @@ void QWorkspaceDockView::onClickPlotContour() {
 * Create a contour plot from the selected workspace group
 */
 void QWorkspaceDockView::showContourPlot() { m_mantidUI->showContourPlot(); }
+
+bool QWorkspaceDockView::executeAlgorithmAsync(Mantid::API::IAlgorithm_sptr alg,
```

Comment:
I think this could use a comment. Seeing an `execAlgorithm` method on a `view` is a little startling at first (but you have vaild reasons for doing it as discussed previously)